x-pack/plugins/synthetics/server/legacy_uptime/lib/saved_objects/migrations.test.ts

```diff
@@ -91,3 +91,75 @@ describe('add820Indices migration', () => {
     });
   });
 });
+
+describe('remove890Indices migration', () => {
+  const context = { log: { warning: () => {} } } as unknown as SavedObjectMigrationContext;
+
+  const makeSettings = (heartbeatIndices: string): SavedObject<DynamicSettingsAttributes> => {
+    return {
+      id: '1',
+      type: 't',
+      references: [],
+      attributes: {
+        heartbeatIndices,
+        certAgeThreshold: 1,
+        certExpirationThreshold: 2,
+        defaultConnectors: ['example'],
+      },
+    };
+  };
+
+  it("removes the synthetics-* index if it's in the indices settings", () => {
+    const doc = makeSettings('heartbeat-8*,something_else');
```

Comment:
This test doesn't actually contain `synthetics-*` so the description seems off.
```suggestion
    const doc = makeSettings('heartbeat-8*,synthetics-*,something_else');
```